src/kata4.humanCatDogYears.js

```diff
@@ -1,3 +1,22 @@
-const humanCatDogYears = (number) => {};
+const humanCatDogYears = (number) => {
+	const human = number;
```

Comment:
Theres no need to store this value, its already in the `number` variable so theres no point if redeclaring it like this. You can just return `number` for the first element in the array on line 19